docs/Telemetry.md

```diff
@@ -106,6 +106,15 @@ _Subscribing Event_:
     }, []);
 ```
 
+### Broadcast Events
+
+Refer to the below table to understand different broadcast events raised during the lifetime of the chat widget.
```

Comment:
Please add another line saying "These events can be subscribed to perform certain optional features of Live Chat Widget"